sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/IncrementalExecution.scala

```diff
@@ -212,6 +212,7 @@ class IncrementalExecution(
           case stateStoreWriter: StateStoreWriter =>
             val metadata = stateStoreWriter.operatorStateMetadata()
             // TODO: [SPARK-48849] Populate metadata with stateSchemaPaths if metadata version is v2
+            // https://github.com/apache/spark/pull/47273
```

Comment:
Sorry, I meant Github PR comment. This code is going to be "committed" and the PR is "proposal". Different level of confidence. Please remove these PR links from code but leave them as Github PR comment.